### PR TITLE
Timer graphics adjustments

### PR DESCRIPTION
**Pull request by nicolasdellatorre**
During a turn in the timer slide there is the player icon and "Turno" has become "Turno di gioco". How can I adjust the buttons?

### CODE REVIEW

**Comment by adolfintel** [comment]
Can you explain the problem better? In italian is fine too

**Comment by nicolasdellatorre** [comment]
Nella lista dei miglioramenti da fare c'è la voce "redesign della schermata del timer: sistemare l'ordine dei pulsanti". Cosa si intende? Grazie

**Comment by adolfintel** [comment]
Era solo una cosa estetica e per chiarezza, aveva fatto una bozza Matteo, appena lo sento me la faccio dare

**Comment by nicolasdellatorre** [comment]
Following Matteo's instructions: there is a differentiation between the name of the player and that of the character, the player's team is more clearer (even in the team timer) and the title attribute has been added to each button so that when passing over it with the mouse its function is clear.

File: CiarnuroRT/webapp/ciarnurort.js [hunk 1]
```diff
@@ -1388,8 +1388,11 @@ setInterval(function(){
                         e.teamTime=getPlayersByTeamId(e.player.team).length*timePerPlayerMs; //for new team
                     }
                 }else I("teamTContainer").style.display="none";
-                I("playerAndCharacterName").textContent=e.player.playerName+" - "+e.player.characterName;
-                I("gtTeamName").textContent=e.player.team===0?"Solo":getTeamById(e.player.team).teamName;
+                let profilePic=document.getElementById('profilePic');
+                profilePic.src="pics/races/" + e.player.race +""+e.player.sex+ ".png";
+                I("playerAndCharacterName").innerHTML=e.player.playerName+"<br>"+"<span style='font-style:italic;'>"+e.player.characterName+"</span >";
```
Comment (adolfintel) [inline CiarnuroRT/webapp/ciarnurort.js]: Don't use br, use margin-top in CSS

File: CiarnuroRT/webapp/index.html [hunk 2]
```diff
@@ -116,8 +116,8 @@
         <div class="slide invisible" id="gameTimer">
             <div class="upper">
                 <div class="content">
-                    <div class="title">Turno <span id="roundNumber"></span> - <span id="gtDate"></span></div>
-                    <div class="banner"><span id="playerAndCharacterName"></span><br/><span id="gtTeamName"></span></div>
+                    <div class="title">Turno di gioco <span id="roundNumber"></span> - <span id="gtDate"></span></div>
+                    <div style="text-align:center;font-size:1.4em;margin-bottom: 0.2em;" id="playerInfo"><img id="profilePic" style="position:relative;left:0.5em;border-radius:50%;height:4em;width:4em;border:0.2rem solid #404040;"/><br><span id="playerAndCharacterName" style="font-weight:bold;"></span><br/><span id="gtTeamName"></span></div>
```
Comment (adolfintel) [inline CiarnuroRT/webapp/index.html]: Inline CSS should only be used for small variations of the style, move this style in the CSS file

**Review by adolfintel — CHANGES_REQUESTED** [review]
Looks good, Matteo just checked it too. I suggested a couple of improvements to the code.

**Pull request merged**